test/e2e/framework/framework.go

```diff
@@ -373,6 +374,11 @@ func (f *Framework) CreateNamespace(baseName string, labels map[string]string) (
 	if ns != nil {
 		f.namespacesToDelete = append(f.namespacesToDelete, ns)
 	}
+
+	if !f.SkipDefaultPSPBinding {
+		CreateDefaultPSPBinding(f, ns.Name)
```

Comment:
granting superuser permissions inside every e2e namespace wasn't exactly what I expected. is this an interim step? if so, add a TODO with the plan for scoping this down the tests that require elevated permissions.

@deads2k any feedback based on the process you went through to enable RBAC in e2e?

Reply:
So to summarize
1. enabling PSP in the test env would ensure the addons configured at startup worked with the OOB config. That's good. 
2. The default e2e behavior would grant privileged PSP access to any e2e test namespace that didn't opt out
    - this depends on a well-known or e2e-created privileged PSP… I'd lean toward e2e-created to remove dependencies on cluster config
    - That seems reasonable, at least as a first step, though I'd be interested in a follow up that made e2e tests opt in to elevated pod permissions (similar to how they have to opt into RBAC permissions for pods that make API calls). That will help uncover more gaps, usability issues, and assumptions made in our test images
3. The specific PSP e2e test would opt out of privileged access and exercise PSP limitations. 
    - this depends on the cluster config not giving access to privileged PSP use in arbitrary namespaces by default, which is a good thing to expect